ui/src/backoffice/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { connect, RootStateOrAny } from 'react-redux';
 
 import DashboardPageContainer from './containers/DashboardPageContainer/DashboardPageContainer';
-import SearchPageContainer from './containers/SearchPageContainer/SearchPageContainer';
+// import SearchPageContainer from './containers/SearchPageContainer/SearchPageContainer';
```

Comment:
You forgot to remove this comment (in fact this import has been moved to L20 now)